tests/saitests/py3/sai_qos_tests.py

```diff
@@ -436,8 +436,8 @@ def runTest(self):
             # queue 3/4  1                 1               1                                                1                                         1                         # noqa E501
             # queue 5    1                 1               1                                                1                                         1                         # noqa E501
             # queue 7    0                 1               1                                                1                                         1                         # noqa E501
-            # LAG ports can have LACP packets on queue 0, hence using >= comparison
-            assert (queue_results[QUEUE_0] >= 1 + queue_results_base[QUEUE_0])
+            # LAG ports can have LACP packets on queue 7, hence using >= comparison
```

Comment:
Please move the comment to line 459, before checking the counter in Queue 7, or update the comment to make it more clear. Thanks

Reply:
resoved